proto/internal/temporal/server/api/replication/v1/message.proto

```diff
@@ -213,3 +214,11 @@ message NewRunInfo {
     string run_id = 1;
     temporal.api.common.v1.DataBlob event_batch = 2;
 }
+
+message VerifyVersionedTransitionTaskAttributes {
+    string namespace_id = 1;
+    string workflow_id = 2;
+    string run_id = 3;
+    int64 next_event_id = 4;
```

Comment:
I think I was wrong. The event version might not equal to the version in the transition history. We probably should explicitly send the last event version history item.

Reply:
From offline discussion, we don't need the version here. But after some thought, I think we still need last event version history item to get event version to determine if the non-current branch has the events. The state transition on the verified task may not generate events, so we cannot get the last event version from the state transition failover version.

Reply:
not sure I follow. let's sync offline again.